src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -119,12 +119,23 @@ public async Task<JObject> TryToRunOnLoadedClasses(string varName, CancellationT
                     }
                 }
                 var store = await proxy.LoadStore(sessionId, token);
+                var info = ctx.CallStack.FirstOrDefault().Method.Info;
+                var currentAssembly = info.Assembly;
+                var namespaceName = info.TypeInfo.Namespace;
+                var classNameToFindWithNamespace = namespaceName + "." + classNameToFind;
+                var type = currentAssembly.GetTypeByName(classNameToFindWithNamespace);
```

Comment:
This can be extracted to a local function like:

```csharp
bool TryGetTypeIdFromName(string typeName, out string? typeId)
{
   typeId = null;
   var type = currentAssembly.GetTypeByName(classNameToFindWithNamespace);
   if (type == null)
       return false;

   typeId = await sdbHelper.GetTypeIdFromToken(sessionId, currentAssembly.DebugId, type.Token, token);
   return true;
}
```

Then you can use it in the two places.